### PR TITLE
Basic material theme for toolbar

### LINKED ISSUE #1003: Material theme for toolbar
Material styles toolbars and app top-bars as solid coloured headers with contrasting text and icons.
Details can be found here: https://material.io/develop/web/components/top-app-bar/

We should be able to achieve this by composing classes from `@material/top-app-bar` but we may need to make changes to our existing widget dom structure to support this.

### PR DESCRIPTION
**Pull request by maier49**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [ ] WidgetProperties are exported

**Description:**
Adds basic styling (background color and typography) for the toolbar under the material theme.
Resolves #1003

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:

**Comment by tomdye** [comment]
can you remove the variables.m.css.d.ts files please?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-psgz6vezj.now.sh
* Tests Deployment: https://dojowidgets-jlmm03jar.now.sh

File: src/theme/material/toolbar.m.css [hunk 1]
```diff
@@ -1,2 +1,10 @@
+@import './variables.css';
+
 .root {
+	background-color: var(--mdc-theme-primary);
```
Comment (tomdye) [inline src/theme/material/toolbar.m.css]: i think there may be some sizing etc to take from the material theme for toolbar here?

File: src/examples/src/widgets/toolbar/HeadingCollapsed.tsx [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-				<Button
+				</RaisedButton>
+				<RaisedButton
 					onClick={() => {
 						icache.set('value', 'Clicked Option C');
 					}}
 				>
 					Option C
-				</Button>
+				</RaisedButton>
```
Comment (tomdye) [inline src/examples/src/widgets/toolbar/HeadingCollapsed.tsx]: why did you change these to raised buttons?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-3npmzov4h.now.sh
* Tests Deployment: https://dojowidgets-9qj9p0cdf.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-ae0j809vl.now.sh
* Tests Deployment: https://dojowidgets-g0dj92ogj.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-3npmzov4h.now.sh
* Tests Deployment: https://dojowidgets-9qj9p0cdf.now.sh
